src/main/java/com/trzewik/spring/domain/game/Game.java

```diff
@@ -244,4 +245,41 @@ public RuntimeException(String msg) {
             super(msg);
         }
     }
+
+    static List<Result> createResults(final Game game) throws Result.Exception {
```

Comment:
all `results` related methods will not be static. And these methods are located in `Game` class - so do you really need put game as parameter of this method?